debian-package/Cargo.toml

```diff
@@ -13,9 +13,7 @@ name = "debian_package"
 path = "src/lib.rs"
 
 [dependencies]
-modality-probe-cli = { path = "../modality-probe-cli" }
 modality-probe-capi = { path = "../modality-probe-capi" }
-modality-probe-udp-collector = { path = "../modality-probe-udp-collector" }
```

Comment:
Is there any way to keep these deps?
This is how `cargo-deb` automatically produces the Debian package `Depends` (runtime dependencies).
I think this change would require us to manage the `Depends` manually.